### DISCUSSION TITLE
Screenshots

### DISCUSSION THREAD

**Discussion by longusnickus** [Heroic-Games-Launcher/HeroicGamesLauncher#254] [Ideas]
since some games freeze if you ALT+TAB and you cannot use the PRINT KEY  a screenshot button would be great

**Comment by adityaruplaha** [comment]
Sorry, this is out of scope for Heroic. Requesting to close...

@flavioislima

**Reply by flavioislima** [reply]
Yeah, try using a different tools for screenshot, one that can take the screenshot and do not need to be opened, like flameshot or something like it.
